setup.py

```diff
@@ -1,10 +1,27 @@
 #!/usr/bin/env python
 from setuptools import find_packages, setup
+from setuptools.command.build_ext import build_ext
 
 long_description = (
     open("README.rst").read() + '\n\n' + open("HISTORY.rst").read()
 )
 
+class BuildClibBeforeExt(build_ext):
```

Comment:
@Lukasa or if you prefer, I could get rid of `super()` and call `build_ext.run(self)` directly.